.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -43,3 +50,10 @@ body:
         issue. If possible, make a PR with a new, failing test to give us a
```

Comment:
How about:

In **bold** near the top: Note that a reproducer in a PR is much more likely to get immediate attention!

Near the top so that people think about their decision bug vs PR immediately.

Then repeat that sentence here, followed by: Failing tests in our CI give us the best starting point for working out fixes.

I could send a PR, plmk.

Reply:
You can add to this one, I expect?

Reply:
Done (0f5bc5820f871c833b0a4614829c7592c2bb2ad2).

Reply:
Sorry that one had an accident. Fixed.